client/src/api/connectJSON.js

```diff
@@ -59,7 +58,7 @@ export async function deleteWholeWeek() {
 }
 
 export async function checkShoppingList() {
-  const shoppingResponse = await fetch("/api/shoppingList");
+  const shoppingResponse = await fetch("/api/shoppingItems");
   const listStatus = await shoppingResponse.json();
   if (listStatus) {
     return listStatus;
```

Comment:
```suggestion
  return listStatus ? listStatus : false
```
and do you really need the `false` in the return-statement?

Reply:
thanks, good point!
Yes I actually do, for a toggle Icon that switches on true or false ;)